File: patterns/javascript-patterns/pattern125.js
// Pattern 125: ...

// *****
//   ***
//     *
//   ***
// *****

const n = 3;
let s1 = n * 2 - 1, s2 = 3;

for (let i = 1; i <= n; i++) {
    let row = '';
    for (let j = 1; j < i; j++) {
        row += '  ';
    }
    for (let k = 1; k <= s1; k++) {
        row += '*';
    }
    s1 -= 2;
    console.log(row);
}

for (let i = 1; i <= n - 1; i++) {
    let row = '';
    for (let j = n - 2; j >= i; j--) {
        row += '  ';
    }
    for (let k = 1; k <= s2; k++) {
        row += '*';
    }
    s2 += 2;
    console.log(row);
}

File: patterns/python-patterns/pattern125.py
# Pattern 125: ...

# *****
#   ***
#     *
#   ***
# *****

n = 3
s1 = n * 2 - 1
s2 = 3

for i in range(1, n + 1):
    print("  " * (i - 1), end="")
    print("*" * s1)
    s1 -= 2

for i in range(1, n):
    print("  " * (n - i - 1), end="")
    print("*" * s2)
    s2 += 2
